doc_files/service_tutorial.md

```diff
@@ -24,32 +24,59 @@ Deploy the bookinfo application to a new namespace. In particular, we create the
 kubectl -n $NAMESPACE apply -f https://raw.githubusercontent.com/iter8-tools/iter8-controller/v0.2.0/doc/tutorials/istio/bookinfo/bookinfo-tutorial.yaml -f https://raw.githubusercontent.com/iter8-tools/iter8-controller/v0.2.0/doc/tutorials/istio/bookinfo/service/productpage-v1.yaml
 ```
 
+## Step 2: Configure Traffic to the Application
+
 Create an Istio gateway for the external host `productpage.example.com`:
 
 ```bash
 kubectl -n $NAMESPACE apply -f https://raw.githubusercontent.com/iter8-tools/iter8-controller/v0.2.0/doc/tutorials/istio/bookinfo/service/bookinfo-gateway.yaml
 ```
 
-******
-Note that at this point, the application is not actually accessible because no Istio `VirtualService` is defined. This tutorial shows how iter8 will create the `VirtualService` for us. This is a use case that might apply, for example, when we do not specify all the Istio components in an application such as when using `helm`.
-
-## Step 2: Create an iter8 Canary Experiment
+At this point, the application is not actually accessible to users because no `VirtualService` has been defined. Rather than manually define it, this tutorial shows how iter8 can be used to create it for us. To do so, define an iter8 canary experiment from the current version of the application to itself. Clearly, this will succeed, and, as a side effect, a `VirtualService` will be created.
 
-We can now create an iter8 `Experiment` specifying using the Kubernetes services. The `targetService` portion of the `Experiment` look like this:
+In the experiment, the `targetService` will look like:
 
 ```yaml
 targetService:
     kind: Service
     baseline: productpage-v1
-    candidate: productpage-v2
+    candidate: productpage-v1
     hosts:
       - name: productpage.example.com
         gateway: productpage-service
 ```
 
-It identifies the type of the baseline and candidate as services using `kind: Service`. Finally, it identifies the external host name and the Istio `Gateway` already configured to route traffic.
+It identifies the type of the baseline and candidate as services using `kind: Service`. The baseline and candidate names are the same. Further, it identifies the external host name and the Istio `Gateway` already configured.
+
+To optimize the bootstrapping process, we can eliminate all of the `successCriteria`. This has the further benefit of eliminating the need for user traffic. We can also alter the  `trafficControl` options to reduce the time and number of iterations required.
+
+You can apply an optimized `Experiment` using:
+
+```bash
+kubectl -n $NAMESPACE apply -f https://raw.githubusercontent.com/iter8-tools/iter8-controller/v0.2.0/doc/tutorials/istio/bookinfo/service/bootstrap-productpage.yaml
+```
 
-You can create the `Experiment` using:
+You can verify that the `Experiment` has been created and finishes quickly:
+
+```bash
+kubectl -n $NAMESPACE get experiment productpage-v2-rollout
```

Comment:
The name of the experiment is actually `productpage-bootstrap`. Please, fix the command above.